index.html

```diff
@@ -205,20 +205,18 @@ <h2>Scope</h2>
-          These issues generally relate to allowing more natural JSON idioms to be expressed using JSON-LD:
+          These issues generally relate to allowing more natural JSON idioms to be expressed using JSON-LD.
         </p>
-        <ul>
-          <li>Allow contexts to be defined within term definitions (Scoped Contexts) that apply for values of that term (<a href="https://github.com/json-ld/json-ld.org/pull/445">PR 445</a>)</li>
-          <li>Index values based on <code>@id</code> and <code>@type</code> (<a href="https://github.com/json-ld/json-ld.org/pull/449">PR 449</a>).</li>
-          <li>Allow for deeper nesting of properties and values without introducing semantics (<a href="https://github.com/json-ld/json-ld.org/pull/451">PR 451</a>).</li>
-          <li>Require <code>@version</code> announcement to enable new features (<a href="https://github.com/json-ld/json-ld.org/pull/473">PR 473</a>).</li>
-          <li>Improve <code>@container</code> description to ensure values always take an array form (<a href="https://github.com/json-ld/json-ld.org/pull/503">PR 503</a>).</li>
-          <li>Allow named graphs to be supported as values without the use of the <code>@graph</code> keyword (<a href="https://github.com/json-ld/json-ld.org/pull/549">PR 549</a>).</li>
-          <li>Allow all values to be used in a map, through the use of a transparent <code>@none</code> index (<a href="https://github.com/json-ld/json-ld.org/pull/569">PR 569</a> – pending).</li>
-        </ul>
-        <p>Additionally, changes have been made to align with updated dependent specifications and
-          to separate the JSON-specific syntax from a more abstract representation compatible
-          with binary and other forms closely related to JSON (e.g., <a href="https://tools.ietf.org/html/rfc7049">CBOR</a> and <a href="http://yaml.org/spec/1.2/spec.html">YAML</a>, see <a href="https://github.com/json-ld/json-ld.org/pull/485">PR 485</a>).</p>
+        <p>The <a href="" class="todo">JSON-LD Working Group</a> will work to address the needs
+          of the community as expressed through <a href="https://lists.w3.org/Archives/Public/public-linked-json/">email</a> and <a href="https://github.com/json-ld/json-ld.org/issues?utf8=✓&q=is%3Aissue+created%3A%3E2014-01-01+-label%3Awebsite+-label%3Aplayground+">issues</a> along with other
+          emergent considerations of stakeholders for expressing data using JSON
+          and related data formats that are consistent with the general objectives
+          for describing data on the World Wide Web as expressed in
+          <a href="https://www.w3.org/TR/dwbp/">Data on the Web Best Practices</a>.
+          The group will further the Application Programming Interface to include
+          Framing, Filtering and other transformation interfaces.</p>
+        <p>A summary of changes addressed by the <a href="https://www.w3.org/community/json-ld/">JSON for Linking Data Community Group</a>
```

Comment:
I would rather say "proposed" by the CG (for the purposes of this charter). It makes the relationship between the two groups cleaner.